http-verbs-common/src/main/scala/uk/gov/hmrc/http/HttpReadsInstances.scala

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.http
+
+import play.api.libs.json.{JsValue, JsError, JsResult, JsSuccess, Reads => JsonReads}
+import scala.util.{Failure, Success, Try}
+
+trait HttpReadsInstances extends HttpReadsHttpResponse with HttpReadsEither with HttpReadsOption with HttpReadsJson
+
+object HttpReadsInstances extends HttpReadsInstances
+
+import HttpReadsInstances._
+
+trait HttpReadsHttpResponse {
+  /** returns the HttpResponse as is - you will be responsible for checking any status codes. */
+  implicit val readRaw: HttpReads[HttpResponse] =
+    HttpReads.ask.map { case (_, _, response) => response }
+}
+
+trait HttpReadsEither {
```

Comment:
I don't understand the purpose of this. I would have thought this was to give us all the `HttpReads[Either[X, Y]]` variants but we have Eithers returned in HttpReadsJson too. It's a bit confusing

Reply:
Good catch! We don't need to repeat the Eithers in HttpReadsJson - since they are composable.